mapeditor/validator.h

```diff
@@ -30,13 +31,18 @@ class Validator : public QDialog
 		bool critical;
 		
 		Issue(const QString & m, bool c): message(m), critical(c) {}
+
+		bool operator <(const Issue & other) const
+		{
+			return message < other.message && critical == other.critical;
```

Comment:
let's drop the flag check

```suggestion
			return message < other.message;
```